README.md

```diff
@@ -122,5 +122,7 @@ However, such problems are best solved with social norms, not with software
 licenses. Any attempt to complicate the software license of Zig would risk
 compromising the value Zig provides to users.
 
-Therefore, Zig is available under the MIT (Expat) License, and comes with a
-humble request: use it to make software better serve the needs of end users.
+Therefore, Zig is available under the MIT (Expat) License with the exception
+of macOS libc headers which are licensed under Apple Public Source License Version 2.0.
```

Comment:
glibc is LGPL

Reply:
That's a good point, thanks. Maybe while we're here we should add the missing exceptions to the master tree? @andrewrk what do you think?

Reply:
FWIW I'm happy to make adjustments either in this or a subsequent PR if we want to keep the PR history clean.

Reply:
IMO just leave this alone for the PR, I'll do a follow up pass on the readme when merging, I want to rewrite that section entirely

Reply:
SGTM! Lemme revert the README changes then.